app/views/groups/index.html.erb

```diff
@@ -0,0 +1,34 @@
+<section class="flex flex-col mb-auto w-full content-center  justify-center ">
+<nav class="flex items-center justify-between bg-customBlue rounded-t-lg h-12 p-3 text-white">
+    <i class="fa-solid fa-bars"></i>
+  <h2 class="text-xl font-semibold">Categories</h2>
+    <i class="text-lg fa-solid fa-magnifying-glass"></i>
+</nav>
+ 
+<div class="w-full h-[595px] max-w-md p-5 bg-zinc-50 border-gray-200 rounded-lg shadow dark:bg-white">
+   <div class="flow-root">
+        <ul role="list" class="divide-y divide-gray-200 flex flex-col gap-3">
+ <% @groups.each do |group| %>
+ <a class="flex items-center bg-white border border-gray-200 rounded-sm shadow  h-[80px] " href="<%= group_path(group) %>">
+            <li class="py-3 sm:py-4">
```

Comment:
- [ ] As per this [requirement](https://github.com/microverseinc/curriculum-rails/blob/main/capstone/rails_capstone.md#:~:text=When%20the%20user%20clicks%20(or%20taps)%20on%20a%20category%20item%2C%20the%20application%20navigates%20to%20the%20transactions%20page%20for%20that%20category.), When the user clicks (or taps) on a category item, the application navigates to the transactions page for that category, not for the category show page as you did. 
For your naming scheme, this link should redirect to `group_operations_path` which is `operations#index` page, not to `group_path` which is `groups#show` page as you did. Please fix that.